README.md

```diff
@@ -143,17 +143,20 @@ There are several command line options to start the daemon:
 Currently you would likely want to start the daemon like this:
 
 ```bash
-cargo run -- -r http://knilxof.org:4242 --disable-tls
+./run.sh -- -r http://knilxof.org:4242 --disable-tls
```

Comment:
yes, it's merely an alias to make things easier for the developer now that we have several binaries.

Totally open to rework this though :)